### ISSUE TITLE
Starred messages seem to get marked as read and not move up in the email stack.

### ISSUE THREAD

**Issue by JustYourAverageGithubUser** [CTemplar/webclient#1640]
### Your environment

Firefox 95.0.2 (64-bit)
Ctemplar Webclient

### Steps to reproduce

1. Mark any email thread as starred.
2. reply to the email chain from the non-ctemplar address
3. Email number will update but email will appear to be read and not move up in the stack of emails, as if no one ever replied to an email.

Obviously, this is counterproductive to the whole point of using the star system (marking emails as important.) I have just figured out about this bug because I missed some important emails due to having no idea anyone had replied to them.

### Expected behavior

Starred email shows up in both unread in the starred folder and unread in whatever folder you have it sorted to go into. Properly stacking to the very top of your email list, as all unread emails do.

### Actual behavior

Does not do any of that and acts as if the email has never been replied to. It also appears to make it so even after the email thread has the star removed, new replies still automark as read and do not properly update when a new email is added to the thread.

**Comment by The-Hidden-Hand** [comment]
@PrabhuSuriya @OneWhiteBird Thoughts?

**Comment by OneWhiteBird** [comment]
Starred messages seem to get marked as read and not move up in the email stack.

> Starred messages seem to get marked as read
Unable to reproduce. Please check back with the user.

> and not move up in the email stack.
This is intended behavior. Starring messages are just to signal important messages. However, they are not pins.

**Comment by The-Hidden-Hand** [comment]
Filter settings could be affecting starred messages for the customer.

**Comment by JustYourAverageGithubUser** [comment]
> > and not move up in the email stack.
> 
> This is intended behavior. Starring messages are just to flag important messages. They are not pins, so they respect the regular chronological order.

I know they are not pins, I was saying that starred messages do not move to the top whenever a new reply is added to the chain (something all other emails do.) However, this seems to have been fixed somehow and I'm no longer experiencing the issue despite (more information below.)

> Filter settings could be affecting starred messages for the customer.

I had checked my filter settings before and the issue persisted. However, the issue seems to have fixed itself as of just a few days ago. I have no idea what seemed to have fixed it,  I haven't messed with CTemplar configurations or anything pretty much since making this post

**Comment by JustYourAverageGithubUser** [comment]
I can even reproduce the issue using my other ProtonMail email, sometimes it gets marked as unread properly, other times it does not, but the issue only happens if the email is starred.

I double checked and none of my filters mark emails as read, just move them to certain folders for convenience.

**Comment by The-Hidden-Hand** [comment]
> Emails in this specific thread seem to sometimes mark as unread properly, and other times do not.

I've also experienced this.

**Comment by PrabhuSuriya** [comment]
@nitishkansal the mail unread status issue was fixed recently right?

**Comment by nitishkansal** [comment]
Yes

**Comment by PrabhuSuriya** [comment]
@The-Hidden-Hand What exactly needs to be fixed here form webclient? I believe this would have been fixed by Nitish recent fix. (Anyway mail order and read status are driven by backend)

**Comment by JustYourAverageGithubUser** [comment]
> @The-Hidden-Hand What exactly needs to be fixed here form webclient? I believe this would have been fixed by Nitish recent fix. (Anyway mail order and read status are driven by backend)

I just received 2 emails in the thread just today and neither of them were marked as unread, so I never knew I received them. So, it's not fixed.

**Comment by nitishkansal** [comment]
I just double checked it and marked a conversation as starred and marked it read. I received a reply on the thread and it was marked as unread and showing on the top as it was the latest message in my mail list. Can you please give us steps to reproduce the issue?

**Comment by JustYourAverageGithubUser** [comment]
> I just double checked it and marked a conversation as starred and marked it read. I received a reply on the thread and it was marked as unread and showing on the top as it was the latest message in my mail list. Can you please give us steps to reproduce the issue?

The steps to reproduce my issue have not changed. @The-Hidden-Hand has even managed to reproduce the issue as mentioned above.

**Comment by JustYourAverageGithubUser** [comment]
So, I tried starting a new email thread with the individual in question, and the emails still mark as read.
I double-checked all my filters and not a single one of them marks anything as read, only moves emails to a different folder.

They are using gmail. Did somehow starring them cause their whole contact profile to be broken? I have no idea why CTemplar would do this. It's at the point where the only email that matters (a business email) is not functioning in a way where I can reply on time.

**Comment by JustYourAverageGithubUser** [comment]
This issue continues to confuse me. I went and tried to remove the contact from my contacts list (assuming that maybe something had messed up.) Still, every email they send me gets automatically marked as read and the only thing that changes is the number of replies the email thread has. I'm having this issue with no other email and exclusively emails from this person, which started after I starred their message.

Is there any way I could further assist in trying to help fix this bug?
